stock_operating_unit/security/stock_security.xml

```diff
@@ -19,7 +19,8 @@
 
     <record id="ir_rule_stock_picking_type_allowed_operating_units" model="ir.rule">
         <field name="model_id" ref="stock.model_stock_picking_type"/>
-        <field name="domain_force">['|',
+        <field name="domain_force">['|', '|',
+            ('warehouse_id','=', False),
```

Comment:
This means that, if a record of the type ```stock.picking.type``` has no warehouse set, then you can read the record. Is this what you want? Otherwise, I'd say you wanted instead:
```xml
<field name="domain_force">['&', '|',
    ('warehouse_id','=', False),
    ('warehouse_id.operating_unit_id','=', False),
    ('warehouse_id.operating_unit_id','in',user.operating_unit_ids.ids)]
</field>
```

Reply:
Exactly that is what I want, cause currently if a warehouse is not set you can't read the record.